aiohttp/client.py

```diff
@@ -406,6 +417,9 @@ async def _request(
         except ValueError as e:
             raise InvalidURL(str_or_url) from e
```

Comment:
This lacks specificity slightly, `InvalidURL` is a base now:
```suggestion
            raise 
InvalidUrlClientError(str_or_url) from e
```